src/main/java/com/shopify/model/MetafieldValueType.java

```diff
@@ -12,7 +15,11 @@ private MetafieldValueType(final String value) {
 	}
 
 	public static MetafieldValueType toEnum(final String value) {
-		if (STRING.toString().equals(value)) {
+		if (SINGLE_LINE_TEXT_FIELD.toString().equals(value)) {
+			return MetafieldValueType.SINGLE_LINE_TEXT_FIELD;
+		} else if (NUMBER_INTEGER.toString().equals(value)) {
+			return MetafieldValueType.NUMBER_INTEGER;
+		} else if (STRING.toString().equals(value)) {
```

Comment:
String is no longer supported. I think to be able to gracefully handle if someone gives us string as the type, then would to be just return SINGLE_LINE_TEXT_FIELD.

Reply:
Just clarifying the String should NOT be part of the enum anymore, but they should be able to pass us the value of string and receive SINGLE_LINE_TEXT_FIELD back